java/connector-node/risingwave-sink-dynamic/src/main/java/com/risingwave/connector/sinkwriter/SinkFunctionImpl.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 RisingWave Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.risingwave.connector.sinkwriter;
+
+import org.apache.flink.streaming.api.functions.sink.SinkFunction;
+import org.apache.flink.table.data.RowData;
+
+/*
+ * The SinkFunction-type sink will invoke this class to write RW data to the downstream.
```

Comment:
The javadoc is too simple. For all the wrapper adapters, we may have more detailed java doc. It may includes:
* the key methods of the wrapped interface.
* how and when these key methods are called in flink, and how we hack the interface in RisingWave.